<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphTest</class>
 <widget class="QWidget" name="GraphTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>6</y>
     <width>480</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_value">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 22px;</string>
   </property>
   <property name="text">
    <string>X value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_x_min">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_x_max">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_y_min">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Y min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_y_max">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Y max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_draw">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>320</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gray;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pushButton_step">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
</string>
   </property>
   <property name="text">
    <string>step</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
